api/proxy/docs/help_out.txt

```diff
@@ -25,6 +25,13 @@ GLOBAL OPTIONS:
    
     --arbitrum-da.port value            (default: 3101)                    ($EIGENDA_PROXY_ARB_DA_PORT)
           Server listening port
+   
+    --arbitrum-da.process-invalid-cert  (default: false)                   ($EIGENDA_PROXY_ARB_DA_PROCESS_INVALID_CERT)
+          Whether or not the CustomDA server should return a `CertificateValidationError`
+          to the arbitrum nitro derivation pipeline which "drops" the DA Cert by treating
```

Comment:
is `process-invalid-cert` the right wording here? what is returned when this is false? are both returning 400s just with a different error in the body?

Reply:
technically since it's JSON RPC there isn't a notion of 400 return. both would be returning 200s with a different error body where (`process-invalid-cert=true`) would return a `CertificateValidationError` that the nitro derivation pipeline understands to discard the DA Cert and continue